pipeline/processes/shaping/shaping.C

```diff
@@ -0,0 +1,74 @@
+Int_t shaping() {
+    TRestRawSignalEvent* ev = new TRestRawSignalEvent();
+
+    TRestRawSignal* sgnl = new TRestRawSignal();
```

Comment:
I don't see these two pointers `*ev` and `*sgnl` being deleted, so this can lead to memory leaks...

Please, delete them or move to ´unique_ptr´

Reply:
It is a simple macro, and the `new` will be destroyed when the execution ends. So whats not big issue here.

But, ok, I add the deletes here.